frank/io.py

```diff
@@ -138,6 +142,7 @@ def save_fit(u, v, vis, weights, sol, prefix, save_solution=True,
         File format in which to save the fit's output UVTable(s)
     """
 
+    logging.info('  Saving results')
```

Comment:
This is fine, but could be more explicit about what 'results' are

Reply:
Changed to 'Saving fit result datafiles' ; the exact datafiles depend on the parameter file. A separate logging statement is used when the UVTables are saved.